internal/querynodev2/segments/mock_data.go

```diff
@@ -302,6 +305,8 @@ func GenTestCollectionSchema(collectionName string, pkType schemapb.DataType, wi
 			binVecFieldSchema,
 			pkFieldSchema,
 			fieldArray,
+			float16VecFieldSchema,
```

Comment:
since we use "bf16" and "fp16" in knowhere,
we'd better to use "fp16VecFieldSchema" and "bf16VecFieldSchema" here